eclass/autotools.eclass

```diff
@@ -378,10 +378,19 @@ eaclocal() {
 	# See bug #677002
 	if [[ ! -f "${T}"/aclocal/dirlist ]] ; then
 		mkdir "${T}"/aclocal || die
-		cat <<- EOF > "${T}"/aclocal/dirlist || die
-			${BROOT}/usr/share/aclocal
-			${ESYSROOT}/usr/share/aclocal
-		EOF
+		if [ "${LIBTOOLIZE:-}" = 'slibtoolize' ]; then
+			cat <<- EOF > "${T}"/aclocal/dirlist || die
+				${BROOT}/usr/share/slibtool
+				${ESYSROOT}/usr/share/slibtool
+				${BROOT}/usr/share/aclocal
```

Comment:
Drop these last 2 lines, then unconditionally append them below (so no need for `else`, just always do it underneath: i.e. conditionally add the first 2 slibtool lines, always add the rest after by changing the bottom `>` to >>`).

Reply:
Good idea, done.